cmd/clusterctl/client/move.go

```diff
@@ -36,11 +38,21 @@ type MoveOptions struct {
 	// namespace will be used.
 	Namespace string
 
+	// Directory defines the local directory to store or restore cluster objects from.
+	Directory string
+
+	// RestoreFromFolder restore configuration from folder. Used with Directory.
+	RestoreFromFolder bool
+
+	// BackupToFolder save configuration to folder. Used with Directory.
+	BackupToFolder bool
+
```

Comment:
We should stop using Backup and Restore, so suggesting 
```suggestion
	// FromDirectory defines the directory where move should read the objects to move (instead of reading from a cluster).
	FromDirectory bool

	// ToDirectory defines the directory where move should save the objects to move (instead of moving to a cluster).
	ToDirectory string
```